alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultUserAccessor.java

```diff
@@ -149,25 +158,36 @@ public boolean changeUserEmailAddress(String username, String emailAddress) {
 
     @Override
     public void deleteUser(String userName) throws AlertForbiddenOperationException {
-        Optional<Long> optionalUserId = userRepository.findByUserName(userName).map(UserEntity::getId);
-        if (optionalUserId.isPresent()) {
-            deleteUserById(optionalUserId.get());
+        Optional<UserEntity> optionalUser = userRepository.findByUserName(userName);
+        if (optionalUser.isPresent()) {
```

Comment:
This code in the if statement and beyond is identical to the other delete method. We can probably abstract some of this to another method.